src/value.rs

```diff
@@ -35,11 +43,10 @@ pub enum Value {
   F64(f64),
```

Comment:
I've seen implementations which use both `I64` and `I32`. Basically, everyone else is doing it 😊

I think this is a large design decision that we need more people to weigh in on, so I'm going to open an issue we can collect votes/opinions in.

Also, what's the difference between `isize` and `i64`?